database/v2/schemas/users.model.js

```diff
@@ -28,6 +28,10 @@ module.exports = (sequalize, type) => {
         allowNull: false,
         defaultValue: 'User'
       },
+      resetPasswordToken: {
+        type: type.STRING(255),
+        allowNull: true
```

Comment:
Do we really need this field?

Reply:
Yes, we need it. this field makes our security more strong. Because, if Hacker will catch link after changing the password by User he will be able to get access to a User account.